test/extended/router/scoped.go

```diff
@@ -28,9 +28,10 @@ const changeTimeoutSeconds = 3 * 60
 var _ = g.Describe("[Conformance][Area:Networking][Feature:Router]", func() {
 	defer g.GinkgoRecover()
 	var (
-		configPath = exutil.FixturePath("testdata", "scoped-router.yaml")
-		oc         *exutil.CLI
-		ns         string
+		configPath  = exutil.FixturePath("testdata", "router-common.yaml")
```

Comment:
It looks like this value only gets used from inside the BeforeEach now so maybe it should get set there? (Likewise in unprivileged.go).

Otherwise lgtm

Reply:
fixed